src/main/resources/static/templates/Http3xxStatusCodeBasedInjection/LEVEL_11/blocked.css

```
@@ -0,0 +1,89 @@
+    font-size: 18px;
+    font-weight: bold;
+}
+
+.url-box {
+    margin-top: 10px;
+    padding: 10px;
+    border: 1px solid black;
+    border-radius: 3px;
+    background: #fff;
+    font-size: 14px;
+    word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)